integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/hive/CatalogHiveIT.java

```diff
@@ -1105,4 +1107,54 @@ void testAlterEntityName() {
       Assertions.assertNotNull(table);
     }
   }
+
+  @Test
+  public void testPurgeHiveManagedTable() throws TException, InterruptedException {
+    ColumnDTO[] columns = createColumns();
+    catalog
+        .asTableCatalog()
+        .createTable(
+            NameIdentifier.of(metalakeName, catalogName, schemaName, tableName),
+            columns,
+            TABLE_COMMENT,
+            createProperties(),
+            new Partitioning[] {IdentityPartitioningDTO.of(columns[2].name())});
+    // Directly get table from hive metastore to check if the table is created successfully.
+    org.apache.hadoop.hive.metastore.api.Table hiveTab =
+        hiveClientPool.run(client -> client.getTable(schemaName, tableName));
+    Assertions.assertEquals(MANAGED_TABLE.name().toLowerCase(Locale.ROOT), hiveTab.getTableType());
+    catalog
+        .asTableCatalog()
+        .purgeTable(NameIdentifier.of(metalakeName, catalogName, schemaName, tableName));
+    Boolean existed = hiveClientPool.run(client -> client.tableExists(schemaName, tableName));
+    Assertions.assertFalse(existed, "The hive table should not be exist");
```

Comment:
The HDFS file of the table should also be asserted not existing in `.Trash`. ( BTW, there is an HDFS client in this class, so you can use it directly to access HDFS :)